ts/Dashboards/Components/HTMLComponent.ts

```diff
@@ -236,6 +236,7 @@ class HTMLComponent extends Component {
     public render(): this {
         super.render();
         this.constructTree();
+        this.sync.start();
```

Comment:
Do we need that in HTMLComp?

Reply:
Yes, it won't change anything if we don't have any sync handlers defined anyway, and if someone defines custom ones, they will be registered.

Without this, custom sync in HTMLComponent will not work.